microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/model/OpenApiModelReader.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.eap.qe.microprofile.openapi.model;
+
+import org.eclipse.microprofile.openapi.OASFactory;
+import org.eclipse.microprofile.openapi.OASModelReader;
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.eclipse.microprofile.openapi.models.info.Info;
+
+/**
+ * Generates a base model to be used by the OpenAPI.
+ */
+public class OpenApiModelReader implements OASModelReader {
+
+    /**
+     * Creates a new {@link Info} instance, modifying the output OpenAPI document in order to add local router
+     * information
+     * 
+     * @return
```

Comment:
I think this can be removed.

Reply:
Why? I added a comment for the return value.